fili-core/src/main/java/com/yahoo/bard/webservice/data/DruidHavingBuilder.java

```diff
@@ -68,15 +67,34 @@ public static Having buildMetricHaving(LogicalMetric metric, Set<ApiHaving> havi
     /**
      * Create a list of NumericHavings for the values specified and OR them together.
      *
-     * @param metric  The metric that the operation applied to.
-     * @param having  The ApiHaving object
-     *
+     * @param metric The metric that the operation applied to.
+     * @param having The ApiHaving object
      * @return A single having representing the API Filter
      */
     public static Having buildHaving(LogicalMetric metric, ApiHaving having) {
         LOG.trace("Building having using metric: {} and API Having: {}", metric, having);
 
         HavingOperation operation = having.getOperation();
+
+        Set<Double> values = having.getValues();
```

Comment:
Since order will matter here, we're going to need to change the internals of ApiHaving to use a List, not a Set for it's values list.